==== triton/lib/Dialect/TritonGPU/Transforms/RemoveLayoutConversions.cpp#11 - /google/src/cloud/chenguangwang/mlir_71d50c890bad943ab23ee9b32638b2366351f8f8_1686777421/triton/lib/Dialect/TritonGPU/Transforms/RemoveLayoutConversions.cpp ====
# action=edit type=text
--- triton/lib/Dialect/TritonGPU/Transforms/RemoveLayoutConversions.cpp	2023-06-12 18:03:13.000000000 -0700
+++ triton/lib/Dialect/TritonGPU/Transforms/RemoveLayoutConversions.cpp	2023-06-14 14:23:33.000000000 -0700
@@ -76,7 +76,7 @@
           convert.getLoc(), tmpType, convert.getOperand());
       auto newConvert = rewriter.create<triton::gpu::ConvertLayoutOp>(
           convert.getLoc(), dstType, tmp);
-      rewriter.replaceOp(op, {newConvert});
+      rewriter.replaceOp(op, newConvert);
       return mlir::success();
     }
     return mlir::failure();
